binr/rahash2/rahash2.c

```diff
@@ -72,17 +72,29 @@ static void do_hash_seed(const char *seed) {
 
 static void do_hash_hexprint(const ut8 *c, int len, int ule, int rad) {
 	int i;
-	if (ule) {
-		for (i = len - 1; i >= 0; i--) {
-			printf ("%02x", c[i]);
+	if (!len) {
+		double e = r_hash_entropy (buf, len);
+		if (rad) {
+			printf ("entropy: %.8f\n", e);
+		} else {
+			printf ("0x%08"PFMT64x "-0x%08"PFMT64x " %.8f: ",
+				from, to > 0? to - 1: 0, e);
+			r_print_progressbar (NULL, 12.5 * e, 60);
```

Comment:
probably this is the reason of travis being red

Reply:
I have a PR on r2r adding a new test for entropy, other tests pass locally.
https://github.com/radare/radare2-regressions/pull/1227